Makefile

```diff
@@ -1,3 +1,7 @@
+VERSION=$$(git describe --tags --always)
```

Comment:
It'd be better to use `?=` instead of only `=`. This will then give the ability to override the value of `VERSION`, something like `VERSION=x.y.z make ...`

Reply:
Great point. I will update this.